model/licensing.py

```diff
@@ -32,7 +26,56 @@ class PolicyException(Exception):
     pass
 
 
-class License(Base):
+class LicenseStatus(PythonEnum):
+    preorder = "preorder"
```

Comment:
Could you explain how a `preorder` license can be created? When it's status becomes `preorder`?

Reply:
It happens when a library purchases a book from the exchange that is not released yet. So they own a license, but its for a preorder, not for the book until the book is released.

Reply:
Thank you for the clarification. Do we need also some code to assign this specific status?

Reply:
Its getting set [here](https://github.com/ThePalaceProject/circulation/pull/66/files#diff-e5370ef0beeb9214e44ff691f352493af2a4f4f6776092763dde8b1b2ba4bc13R956) in the corresponding PR on circulation.